runtime/nls/cfre/cfrerr.nls

```diff
@@ -1297,3 +1297,20 @@ J9NLS_CFR_ERR_BAD_NEST_TOP_INDEX.user_response=Contact the provider of the class
 
 # END NON-TRANSLATABLE
 
+# <clinit> method at version 51.0 or above does not have static flag set
+J9NLS_CFR_ERR_CLINIT_NOT_STATIC=Method <clinit> is not static
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_CLINIT_NOT_STATIC.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_CLINIT_NOT_STATIC.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_CLINIT_NOT_STATIC.user_response=Contact the provider of the classfile for a corrected version.
+
+# END NON-TRANSLATABLE
+
+# <clinit> method at version 51.0 must be void and take no arguments
```

Comment:
Where is this used?  The staticverify.c change was reverted before we opened the code.  Did it get lost?

Reply:
Yes. I've added it back in. I will rebuild this change.